com.js8call.JS8Call.yaml

```
@@ -0,0 +1,39 @@
+app-id: com.js8call.JS8Call
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: js8call
+rename-desktop-file: js8call.desktop
+finish-args:
+  - --device=all
+  - --share=network
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+modules:
+  - shared-modules/linux-audio/fftw3f.json
+  - shared-modules/libusb/libusb.json
+
+  - name: hamlib
+    sources:
+      - type: archive
+        url: https://github.com/Hamlib/Hamlib/releases/download/4.5.5/hamlib-4.5.5.tar.gz
+        sha256: 601c89f32ed225e9527ade3d64d0d05d23202c05ae21ffa77e59d70ee4597fcd
+        x-checker-data:
+          type: anitya
+          project-id: 1292
+          url-template: https://github.com/Hamlib/Hamlib/releases/download/$version/hamlib-$version.tar.gz
+
+  - name: js8call
+    buildsystem: cmake-ninja
```

```suggestion
    buildsystem: cmake
```

Since it seems to choke on Fortran code, this should help.